cloud

```diff
@@ -2299,6 +2305,7 @@ function execute() {
     declare test="false"
     declare log_level="INFO"
     declare optimize=""
+    declare cache_bucket_name=""
```

Comment:
nit; Move this between `log_level` and `optimize`, just so we can keep it uniform with the rest of the code

Reply:
Actually, based on https://github.com/kernelci/kcidb/pull/439/files#r1273098653 just

```suggestion
```

Reply:
okay